internal/repos/github_test.go

```diff
@@ -718,6 +786,93 @@ func TestRepositoryQuery_Do(t *testing.T) {
 	}
 }
 
+func TestGithubSource_SearchRepositories(t *testing.T) {
+	assertAllReposListed := func(want []string) typestest.ReposAssertion {
+		return func(t testing.TB, rs types.Repos) {
+			t.Helper()
+
+			have := rs.Names()
+			sort.Strings(have)
+			sort.Strings(want)
+
+			if !reflect.DeepEqual(have, want) {
+				t.Error(cmp.Diff(have, want))
+			}
```

Comment:
nit: 

- Is there a reason to use reflect as opposed to just expecting empty result of `cmp.Diff`?
- nit-nit: usually `cmp.Diff(want, have)` is more readable than `cmp.Diff(have, want)` since the right hand side is rendered with `+` and left hand side with `-`.

```suggestion
			if diff := cmp.Diff(want, have); diff != "" {
				t.Error(diff)
			}
```

Reply:
updated